plugins/notifications/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set(ubuntu_notifications_panel_SRCS
     click_applications_notify_model.cpp
     )
 
+configure_file(qmldir.in qmldir)
```

Comment:
Why do we need to configure the file? there is nothing to "configure" its a static file, its only for the "copy to build" dir you want, please set COPYONLY https://cmake.org/cmake/help/latest/command/configure_file.html